### ISSUE TITLE
Staging process fails for buckets in multiple regions

### ISSUE THREAD

**Issue by bernardosrulzon** [snowplow/snowplow#2823]
We've just launched a new Beanstalk instance in São Paulo and I edited config.yml to pull data from both buckets:

```
s3:
    region: us-east-1
        in:
          - s3://elasticbeanstalk-us-east-1-1234/resources/environments/logs/publish/abc
          - s3://elasticbeanstalk-sa-east-1-1234/resources/environments/logs/publish/def
```

The staging process fails when processing the São Paulo bucket:

```
(t6)    MOVE elasticbeanstalk-sa-east-1-1234/resources/environments/logs/publish/def/ghi/_var_log_tomcat8_rotated_local
host_access_log.txt1469476862.gz -> gn-snowplow-etl/processing/var_log_tomcat8_rotated_localhost_access_log.2016-07-25-20.us-east-1.i-ghi.
txt.gz
Problem copying resources/environments/logs/publish/def/ghi/_var_log_tomcat8_rotated_local
host_access_log.txt1469476862.gz. Retrying.
```

Note that Snowplow is trying to move a file from the São Paulo bucket, but still appends `us-east-1` to the filename

**Comment by alexanderdean** [comment]
Hi @bernardosrulzon - right, yes - EmrEtlRunner doesn't support working with S3 buckets across multiple regions currently...

**Comment by bcharp** [comment]
I'm currently upgrading our snowplow stack and it worked well before R83 (at least in our case). In R83, we've got clojure collector in europe and emr, etc.. in US Standard, it doesn't work. I think it made sense to be able to have collector in multiple region and only one pipeline.

**Comment by BenFradet** [comment]
will be fixed by #276
